services/core-api/app/api/now_applications/transmogrify_now.py

```diff
@@ -21,6 +21,7 @@
     "Govt. Action Required": "PEV",
     "Referral Complete": "PEV",
     "Referred": "PEV",
+    "No Permit Required": "PEV",
     None: "PEV",
```

Comment:
Thank you for adding this. (Everything else in this file looks like formatting changes)